sources/subs/MenuCreate.class.php

```diff
@@ -0,0 +1,727 @@
+<?php
+
+/**
+ * This class contains a standard way of displaying side/drop down menus.
+ *
+ * @name      ElkArte Forum
+ * @copyright ElkArte Forum contributors
+ * @license   BSD http://opensource.org/licenses/BSD-3-Clause
+ *
+ * This file contains code covered by:
+ * copyright:    2011 Simple Machines (http://www.simplemachines.org)
+ * license:    BSD, See included LICENSE.TXT for terms and conditions.
+ *
+ * @version 1.1 beta 2
+ *
+ */
+
+/**
+ * This class implements a standard way of creating menus
+ */
+Class Menu_Create
```

Comment:
I'd have to rename the file as well for the autoloader to pick it up.  But really I had no reason for the name other than that was the primary function name.  Be glad to rename it.